### ISSUE TITLE
Elo rating system should be completed.

### ISSUE THREAD

**Issue by 10cheon00** [10cheon00/haleygg_server#21]
It should be completed, however the purpose of elo rating is fading.

After players played game, they will watch their results and statistics. And they find out their rank from elo rating.
Each league might have different ***elo rating system*** by managing K factor. it is same meaning with Season inthe *League of Legends*. 
So player will differently feels about their skill on each league.

However, the period of Proleague or Starleague is too shorter than the *League of Legends*. Player can't get enough matches to find their skill and rank. 

In conclusion, my opinion is that show their ***elo rating*** in one system by integrating all league. 
This idea must be discuss with clan managers.

**Comment by 10cheon00** [comment]
0770bd0 This issue was fixed.
If server receives a request to update Elo with credential, delete whole Elo instance and create new elo data using current PlayerTuple instances.
